cmd/tetra/common/flags.go

```diff
@@ -12,3 +14,13 @@ const (
 	KeyTimeout       = "timeout"        // duration
 	KeyRetries       = "retries"        // int
 )
+
+var (
+	Color         string
+	Debug         bool
+	Output        string
+	Tty           string
+	ServerAddress string
+	Timeout       time.Duration
+	Retries       int
```

Comment:
It looks like some of those vars are unused, could we keep only the var used?

Reply:
sure i will remove unused variable